src/Services/SettingsBase.cs

```diff
@@ -71,7 +84,7 @@ protected T GetSetting<T>(Func<T> defaultValue, [CallerMemberName] string key =
 
             // Null values are never stored in runtime or persistent storage.
             if (fallbackValue is not null)
-                _runtimeStorage[key] = (typeof(T), fallbackValue);
+                _runtimeStorage[key] = new(typeof(T), fallbackValue);
```

Comment:
`IsDirty` is true here by default, meaning the setting will be flushed to disk when `SaveAsync()` is called

Reply:
Oh, I hadn't noticed this. Let's change this, we don't need to flush default settings to disk, since they're always available (and always correct) when in memory.

Reply:
Correction, we _shouldn't_ flush default to disk. The default value doesn't need to be constant; it only needs to be static. You could compute a different default every time, if you really wanted to.